docs/spark_expressions_support.md

```diff
@@ -0,0 +1,477 @@
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied.  See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+# Supported Spark Expressions
+
+### agg_funcs
+ - [ ] any
+ - [ ] any_value
+ - [ ] approx_count_distinct
+ - [ ] approx_percentile
+ - [ ] array_agg
+ - [ ] avg
+ - [ ] bit_and
+ - [ ] bit_or
+ - [ ] bit_xor
+ - [ ] bool_and
+ - [ ] bool_or
+ - [ ] collect_list
+ - [ ] collect_set
+ - [ ] corr
+ - [ ] count
+ - [ ] count_if
+ - [ ] count_min_sketch
+ - [ ] covar_pop
+ - [ ] covar_samp
+ - [ ] every
+ - [ ] first
+ - [ ] first_value
+ - [ ] grouping
+ - [ ] grouping_id
+ - [ ] histogram_numeric
+ - [ ] kurtosis
+ - [ ] last
+ - [ ] last_value
+ - [ ] max
+ - [ ] max_by
+ - [ ] mean
+ - [ ] median
+ - [ ] min
+ - [ ] min_by
+ - [ ] mode
+ - [ ] percentile
+ - [ ] percentile_approx
+ - [ ] regr_avgx
+ - [ ] regr_avgy
+ - [ ] regr_count
+ - [ ] regr_intercept
+ - [ ] regr_r2
+ - [ ] regr_slope
+ - [ ] regr_sxx
+ - [ ] regr_sxy
+ - [ ] regr_syy
+ - [ ] skewness
+ - [ ] some
+ - [ ] std
+ - [ ] stddev
+ - [ ] stddev_pop
+ - [ ] stddev_samp
```

Comment:
These are supported according to https://datafusion.apache.org/comet/user-guide/expressions.html

Reply:
I'll double check that but the `spark_builtin_expr_coverage.txt` shows 
```
Unsupported: Expected only Comet native operators but found Spark fallback
```
for both of them. I'll verify if its a test problem or not

Reply:
I ran the test manually 

```
  test("sttdev") {
    Seq(false, true).foreach { dictionary =>
      withSQLConf("parquet.enable.dictionary" -> dictionary.toString) {
        val table = "test"
        withTable(table) {
          sql(s"create table $table(col int) using parquet")
          sql(s"insert into $table VALUES (1), (2), (3)")
          checkSparkAnswerAndOperator(s"SELECT stddev_pop(col) FROM $table")
        }
      }
    }
  }
```
and it fails `Expected only Comet native operators, but found HashAggregate.`

the physical plan is 
```
== Physical Plan ==
AdaptiveSparkPlan isFinalPlan=false
+- HashAggregate(keys=[], functions=[stddev_pop(cast(col#0 as double))])
   +- Exchange SinglePartition, ENSURE_REQUIREMENTS, [plan_id=118]
      +- CometHashAggregate [col#0], Partial, [partial_stddev_pop(cast(col#0 as double))]
         +- CometScan parquet [col#0] Batched: true, DataFilters: [], Format: CometParquet, Location: InMemoryFileIndex(1 paths)[file:/tmp/stddev], PartitionFilters: [], PushedFilters: [], ReadSchema: struct<col:int>
```

Reply:
@viirya @andygrove @kazuyukitanimura @advancedxy do you guys think this is a sign of the expression not natively supported?

Reply:
Have you enabled Comet shuffle?

Reply:
good point